routing/bicycle_directions.cpp

```diff
@@ -0,0 +1,185 @@
+#include "routing/bicycle_directions.hpp"
+#include "routing/car_model.hpp"
+#include "routing/router_delegate.hpp"
+#include "routing/routing_result_graph.hpp"
+#include "routing/turns_generator.hpp"
+
+#include "geometry/point2d.hpp"
+
+#include "indexer/ftypes_matcher.hpp"
+#include "indexer/index.hpp"
+#include "indexer/scales.hpp"
+
+namespace
+{
+using namespace routing;
+using namespace routing::turns;
+
+class AStarRoutingResultGraph : public IRoutingResultGraph
```

Comment:
The class should be named in accordance with it's behavour, not with it's usage. It can be used with DijkstraRouter too, right? Or with single-directional astar, right? Please, simplify it's name to RoutingResultGrap.h